Source/LibationWinForms/GridView/ProductsGrid.cs

```diff
@@ -122,7 +123,7 @@ private void gridEntryDataGridView_CellContextMenuStripNeeded(object sender, Dat
                     {
                         var dgv = (DataGridView)sender;
                         var text = dgv[e.ColumnIndex, e.RowIndex].FormattedValue.ToString();
-                        InteropFactory.Create().CopyTextToClipboard(text);
+						Clipboard.SetDataObject(text, false, 5, 150);
```

Comment:
Interesting. I jumped though these hoops because I didn't think Clipboard was xplat

Reply:
Yup! It's a gui-only feature though, so you need Avalonia to do it.